swri_geometry_util/include/swri_geometry_util/intersection.h

```diff
@@ -51,6 +51,41 @@ namespace swri_geometry_util
       cv::Vec2d p4,
       cv::Vec2d& c);
 
+  /**
+   * Calculate the instersection between two line segments defined by 4 points.
+   *
+   * In the case of parallel overlapping segments, the intersection point
+   * closest to p1 is returned.
+   *
+   * @param[in]   p1   First point of line segemnt 1.
```

Comment:
Typo in `segemnt`.

Reply:
Fixed typo in this function and the existing function above it.